src/near-bindgen.ts

```diff
@@ -1,72 +1,110 @@
 import * as near from "./api";
 
-export function initialize({ }) {
-    return function (target: Object, key: string | symbol, descriptor: TypedPropertyDescriptor<Function>): void {
-    }
-}
+type EmptyParameterObject = Record<never, never>;
+// type AnyObject = Record<string, unknown>;
+// type DecoratorFunction = (
+//   target: AnyObject,
+//   key: string | symbol,
+//   descriptor: TypedPropertyDescriptor<Function>
+// ) => void;
```

Comment:
Remove or apply these types to functions below?

Reply:
@ailisp Forgot to remove the comments... I'll need to test out proper typing for these functions.
BTW, do these functions have to actually return an empty function, or anything at all. I want to understand this better so I can actually define proper types.